qbft/spectest/tests/messages/marshal_justification_with_full_data.go

```diff
@@ -0,0 +1,41 @@
+package messages
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// MarshalJustificationsWithFullData tests marshalling justifications with full data (should omit it)
+func MarshalJustificationsWithFullData() *tests.MsgSpecTest {
+	ks := testingutils.Testing4SharesSet()
+
+	rcMsgs := []*qbft.SignedMessage{
+		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[1], 1, 2),
+	}
+
+	prepareMsgs := []*qbft.SignedMessage{
+		testingutils.TestingPrepareMessage(ks.Shares[1], types.OperatorID(1)),
+	}
+
+	msg := testingutils.TestingProposalMessageWithParams(
+		ks.Shares[1], types.OperatorID(1), 2, qbft.FirstHeight, testingutils.TestingQBFTRootData,
+		testingutils.MarshalJustifications(rcMsgs), testingutils.MarshalJustifications(prepareMsgs))
+
+	r, _ := msg.GetRoot()
+	b, _ := msg.Encode()
```

Comment:
I think we should hardcode the values. Since the test runs the exact same logic in runtime, if something breaks it will never be caught

Reply:
Yes, I agree. Thanks